src/Blocks/custom/greenhouse/greenhouse.php

```diff
@@ -41,6 +43,15 @@
 		[
 			'formPostId' => $greenhouseFormPostId,
 			'formType' => SettingsGreenhouse::SETTINGS_TYPE_KEY,
+			'formResetOnSuccess' => (bool) !Variables::isDevelopMode(),
```

Comment:
No need to cast this to boolean, as the return value is set to boolean and the strict mode will ensure no type juggling happens 🙂